### PR TITLE
Expand open ai connector (embeddings, chat completion, completion_v2)

### PR DESCRIPTION
**Pull request by PowerRobin**
Added 3 new endpoints to the existing OpenAI indie publisher connector.

### CODE REVIEW

File: independent-publisher-connectors/OpenAI/apiDefinition.swagger.json [hunk 1]
```diff
@@ -80,7 +536,7 @@
           }
         },
         "summary": "GPT3 Completes your prompt",
-        "description": "GPT3 Completes your prompt",
+        "description": "GPT3 Completes your prompt (deprecated by OpenAI - use Completion_New)",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenAI/apiDefinition.swagger.json:539]: @PowerRobin please add the "deprecated" property and set it to true. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/deprecate-operation).

File: independent-publisher-connectors/OpenAI/apiDefinition.swagger.json [hunk 2]
```diff
@@ -18,6 +18,462 @@
...
+                  },
+                  "description": "Returned Completion(s)"
+                }
+              }
+            }
+          }
+        },
+        "summary": "GPT3 Completes your prompt",
+        "description": "GPT3 Completes your prompt",
+        "operationId": "Completion_v2",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenAI/apiDefinition.swagger.json]: @PowerRobin all operationIds should be in Camel Case. please change to "CompletionV2". Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations). please fix all operationIds in the swagger file.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/OpenAI/apiDefinition.swagger.json (hunk 2 above)
Comment (PowerRobin) [reply independent-publisher-connectors/OpenAI/apiDefinition.swagger.json]: Thanks for pointing those out @Amjed-Ayoub 

Made the changes.

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hello @PowerRobin ,
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. 
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by Amjed-Ayoub** [comment]
Hello @PowerRobin ,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Review by Amjed-Ayoub — APPROVED** [review]
Hello @PowerRobin ,
I hope you are doing well.
Congratulations, the deployment of your connector and documentation page was already completed, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program.
Thank you very much for working with us.

**Pull request merged**